src/Endatix.Persistence.SqlServer/README.md

```
@@ -14,22 +14,79 @@ dotnet add package Endatix.Persistence.SqlServer
+### Remove last migration
+
+1. First list your migraitons with `dotnet ef migrations list` to check if the migration you want to remove must be unapplied on your local DB
+
+```bash
+dotnet ef migrations list \
+  --startup-project src/Endatix.WebHost \
+  --project src/Endatix.Persistence.SqlServer \
+  --context <AppDbContext|AppIdentityDbContext>
+```
+
+2. If required update to the migraiton via `dotnet ef database update`
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)